test/proposals/Proposal_0.sol

```diff
@@ -125,13 +125,16 @@ contract Proposal_0 is Proposal {
                 650, // midPoint = 10m50s
                 1800 // auctionDuration = 30m
             );
-            LendingTerm termUSDC1 = new LendingTerm(
+            LendingTerm termUSDC1 = new LendingTerm();
+            termUSDC1.initialize(
```

Comment:
this should happen in the gov proposal to initialize the system

Reply:
I'll create the terms through `LendingTermOnboarding.createTerm()` so there will be no need to call the initialize function